boards/arm/stm32f7/nucleo-144/include/board.h

```diff
@@ -508,5 +523,12 @@
 #define GPIO_ETH_RMII_TX_EN   GPIO_ETH_RMII_TX_EN_2
 #define GPIO_ETH_RMII_TXD0    GPIO_ETH_RMII_TXD0_2
 #define GPIO_ETH_RMII_TXD1    GPIO_ETH_RMII_TXD1_1
+#define GPIO_ETH_MII_COL      GPIO_ETH_COL_1
```

Comment:
Why are you adding an MMI signal to a RMII interface?

The above table does not include this signal. If it is used, it should be in the table, if it is not on the HW is should be removed.

Reply:
@robertobucher - once this passes CI Please rebase and fixup and force push.

`git rebase -i master`
- then in the editor move the commits like that fixes a file under the commit that changed the file in the fist palce and change the `pick` to `f`

That will do the `rebase`

then to `git push -f robertobucher master`

That is the force push

Reply:
I can't exactly understand what you mind... I'm not a git expert, I've forked the NUTTX project on my github page and then I work on a local version on my PC. I've changed the files on my local repository and pushed on my remote repository (https://github.com/robertobucher/incubator-nuttx) and now I try to integrate the modifications.